lib/modules/platform/pr-body.ts

```diff
@@ -28,15 +37,24 @@ export function hashBody(body: string | undefined): string {
   return result;
 }
 
-function isRebaseRequested(body: string | undefined): boolean {
-  return !!body?.includes(`- [x] <!-- rebase-check -->`);
+function isRebaseRequested(body: string): boolean | undefined {
+  const match = prCheckboxRe.exec(body);
+  if (!match) {
+    return undefined;
+  }
+  return match.groups?.checkbox === '[x]';
```

Comment:
```suggestion
  return prCheckboxRe.test(body);
```
this is much easier, see other comment

Reply:
Test won't help me to know it there is no checkbox at all.
If we test for `[x]` we will have only binary info about the checked box. i.e. (true for `[x]`, false for `[ ]` and for no checkbox at all)
but I also need this info in order to "migrate" existing onboarding PRs.

This is quite defensive as we can identify this case if there is no embedded config hash comment, but i prefer checking for both.